modules/hostap/src/supp_api.c

```diff
@@ -802,8 +802,11 @@ static int wpas_add_and_config_network(struct wpa_supplicant *wpa_s,
 				goto out;
 			}
 
-			if (!wpa_cli_cmd_v("set_network %d eap %s", resp.network_id, method)) {
-				goto out;
+			if (method != NULL) {
+				if (!wpa_cli_cmd_v("set_network %d eap %s", resp.network_id,
+						method)) {
```

Comment:
Indentation is off, please place method aligned with the first parameter of the function

Reply:
Thanks, please review again